controller/resource.go

```diff
@@ -199,6 +243,7 @@ func (t *Task) updateServiceToken(serviceName, secretName string) error {
 	if err != nil {
 		return fmt.Errorf("creating envoy token: %s", err)
 	}
+	t.Log.Info("service token created successfully", "id", serviceName)
```

Comment:
```suggestion
	t.Log.Info("service token created successfully", "service", serviceName)
```

For consistency. id has been used as the taskARN elsewhere and service was used for serviceName above